libs/styles/article-card.css

```diff
@@ -5,6 +5,7 @@
   box-sizing: border-box;
   display: inline-block;
   border-radius: 4px;
+  border: 1px solid var(--color-gray-200);
```

Comment:
The comment at the top of the file says that this stylesheet is shared between blocks, so you need to be careful here. Please include test pages for the shared blocks to make sure you're not affecting them or add the code under more specific selectors

Reply:
It doesn't affect the shared blocks. Test pages:
- https://main--blog--adobecom.hlx.page/drafts/naguilar/article-feed?milolibs=contextual-search
- https://main--blog--adobecom.hlx.page/drafts/naguilar/recommended-articles?milolibs=contextual-search